lndr-backend/db/migrations/2018-07-03T10.00.00Z_add_settlement_currency_to_pending_credits.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE pending_credits
+ADD COLUMN settlement_currency CHAR(10);
```

Comment:
:+1: At first I was going to say we needed an `AFTER` constraint here to ensure the column gets added to the appropriate position, but after some research, I see that in PostgreSQL there is no notion of controlled column ordering; just a MySQL idiosyncracy. I love that.